tests/unittest_brain_numpy_ma.py

```diff
@@ -13,41 +13,58 @@
 
 from astroid import builder
 
+parametrize = pytest.mark.parametrize("alias_import", [True, False])
+
 
 @pytest.mark.skipif(HAS_NUMPY is False, reason="This test requires the numpy library.")
 class TestBrainNumpyMa:
     """
     Test the numpy ma brain module
     """
 
-    @staticmethod
-    def test_numpy_ma_masked_where_returns_maskedarray():
+    def _assert_maskedarray(self, code):
+        node = builder.extract_node(code)
+        cls_node = node.inferred()[0]
+        assert cls_node.pytype() == "numpy.ma.core.MaskedArray"
+
+    @parametrize
+    def test_numpy_ma_masked_where_returns_maskedarray(self, alias_import):
         """
         Test that calls to numpy ma masked_where returns a MaskedArray object.
 
         The "masked_where" node is an Attribute
         """
-        src = """
-        import numpy as np
+        import_str = (
+            "import numpy as np"
+            if alias_import
+            else "from numpy.ma import masked_where"
+        )
+        func_call = "np.ma.masked_where" if alias_import else "masked_where"
```

Comment:
```suggestion
        func = "np.ma.masked_where" if alias_import else "masked_where"
```

Its just the function name so I think this is a bit clearer.